src/frontend/src/binder/expr/mod.rs

```diff
@@ -118,6 +123,26 @@ impl Binder {
         .into())
     }
 
+    pub(super) fn bind_in_list(
+        &mut self,
+        expr: Expr,
+        list: Vec<Expr>,
+        _negated: bool,
+    ) -> Result<ExprImpl> {
+        // FIXME: Handle `not in`.
```

Comment:
Thought a while and did not get how to implement `NOT IN` (reusing NOT operator or do it in execution expression info?)

Reply:
* `NOT LIKE` is using `NOT` operator.
* `NOT BETWEEN` constructs conditions in reversed order directly.
* `NOT {TRUE,FALSE,NULL}` use a dedicated enum.

My suggestions is to start simple and optimize when we need to.